### PR TITLE
Mu2eII_SM21: removed root and art files

### PR DESCRIPTION
**Pull request by sophiemiddleton**

### CODE REVIEW

File: Tests/DIOSteps.fcl [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #include "Production/JobConfig/primary/DIOtail.fcl"
 source.firstRun: 501 #TODO
-physics.filters.TargetStopResampler.fileNames : [
-"/pnfs/mu2e/tape/phy-sim/sim/mu2e/MuminusStopsCat/MDC2020k/art/39/6a/sim.mu2e.MuminusStopsCat.MDC2020k.001201_00000000.art"]
-# NOTE: to use on the grid do samweb list-file-locations --schema=root --defname="sim.mu2e.MuminusStopsCat.MDC2020k.art" | cut -f1 and place that file name here
+physics.filters.TargetStopResampler.fileNames : ["root://fndca1.fnal.gov:1094/pnfs/fnal.gov/usr/mu2e/tape/phy-sim/sim/mu2e/MuminusStopsCat/Mu2eIIa2/art/40/89/sim.mu2e.MuminusStopsCat.Mu2eIIa2.001201_00000000.art"]
+
+#"/pnfs/mu2e/tape/phy-sim/sim/mu2e/MuminusStopsCat/Mu2eIIa2/art/40/89/sim.mu2e.MuminusStopsCat.Mu2eIIa2.001201_00000000.art"
```
Comment (kutschke) [inline Tests/DIOSteps.fcl:5]: Two things.  

The title of this commit is gitignore but the commit actually changes a different file.  Please confirm your intentions.

Second.  Is this code in active developement?  If so it's OK to leave in commented out lines for short time.  If not then please remove the commented out lines.

**Review by kutschke — CHANGES_REQUESTED** [review]

**Comment by kutschke** [comment]
@sophiemiddleton I have a comment on this PR that needs to be answered.  I will be on vacation starting this evening and won't be back until Wednesday.  If you would like it merged before then, please answer this soon.

**Comment by kutschke** [comment]
@FNALbuild run build tests

File: Tests/DIOSteps.fcl (hunk 1 above)
Comment (sophiemiddleton) [reply Tests/DIOSteps.fcl:5]: This is an update to the stopped muon file

**Comment by sophiemiddleton** [comment]
Hi Rob - the update you refer to is an update to the stopped muons for the DIO to point to the Mu2eII stops

File: Tests/DIOSteps.fcl (hunk 1 above)
Comment (kutschke) [reply Tests/DIOSteps.fcl:5]: Yes, I see that.  I am asking why does the commit comment say "gitignore" when the commit does not modify the .gitignore file?  We discussed modifying .gitignore  to exclude *.root and *.art files and I was expecting to see that.

**Comment by sophiemiddleton** [comment]
This should now be fixed

**Comment by sophiemiddleton** [comment]
@FNALbuild  run build tests

File: Tests/DIOSteps.fcl (hunk 1 above)
Comment (kutschke) [reply Tests/DIOSteps.fcl:5]: Thanks for updating .gitignore.

**Review by kutschke — APPROVED** [review]

**Pull request merged**